src/@utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const MAX_DECIMALS = 5
```

Comment:
well, this is not a variable, this is a const. It is a const in case we decide to change it (as a developer) but it's not a variable to be changed whenever by who deploys the market.